opentelemetry-distro/src/opentelemetry/distro/__init__.py

```diff
@@ -172,4 +172,4 @@ class OpenTelemetryDistro(BaseDistro):
     """
 
     def _configure(self, **kwargs):
```

Comment:
Could you make a comment in the distro readme all the components that are being used as default?

Reply:
Will do as part of https://github.com/open-telemetry/opentelemetry-python/issues/1606